src/test/java/ca/team2706/frc/robot/subsystems/DriveBaseTest.java

```diff
@@ -46,6 +48,9 @@
     @Mocked
     private Notifier notifier;
 
+    @Mocked(stubOutClassInitialization = true)
+    private Config config;

```

Comment:
Again, don't want to be mocking config class here.

Reply:
Config is now "partially mocked" as described in Section 13 of http://jmockit.github.io/tutorial.html. The reason for this partial mock is that expected results for the test depend on
the values of Config.ROBOT_START_ANGLE and that value could be changed at one point in the future (even right before a match). We don't want to have to change the test every time the parameter is changed. If you have a preferred different approach, let us know.

Reply:
@robertlucyshyn I'm pretty sure that this would be a full replacement mock of Config. Partial mocking doesn't use the `@Mocked` at all, as described: http://jmockit.github.io/tutorial/Mocking.html#partial.,

Reply:
Kyle, I'm confused. I thought I pushed a new version of DriveBase.java without the @Mocked on Config. Doesn't lack of a @Mocked plus this:

```
public void testAbsoluteGyro() {
        new Expectations(Config.class) {{
            Config.getROBOT_START_ANGLE();
            result = 0.0;
        }};
```
mean that Config is being mocked only for the getROBOT_START_ANGLE() method?

Reply:
Yes, that's a partial mock.

Reply:
All mocking of Config has now been removed so this matter is now resolved.